### PR TITLE
Django extensions 1.5.0

### PR DESCRIPTION
**Pull request by astamminger**
Hi @ocefpaf @kwilcox,

here's my recipe for django-extensions==1.5.0. Unfortunately I had to pin the django versions down to ==1.8 and <=1.8 for python3 and python2 respectively to make the tests work (ran them locally). I hope you're fine with that.

Best,
Andreas

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,9 +19,10 @@ requirements:
   run:
     - python
     - six >=1.2
-    - django >=1.7
-    - typing
-
+    # Pinning Django to versions not failing the testsuite
+    - django ==1.8  # [py3k]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Can we do `django <=1.8` only and avoid the pre-selector?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/d/django-extensions/django-extensions-{{ version }}.tar.gz
-  sha256: 1f626353a11479014bfe0d77e76d8f866ebca1bb5d595cb57b776230b9e0eb92
+  sha256: 2e44532500f3720e5b7b4e3ac483fa6e0c5e3feadc132791f08b6ed4ee9ca704
 
 build:
   number: 0
-  noarch: python
```
Comment (ocefpaf) [inline recipe/meta.yaml]: This is not in effect yet b/c we need to re-render but I would like to revert this change if you can avoid the pre-seletor in https://github.com/conda-forge/django-extensions-feedstock/pull/33/files#r221308074

File: recipe/meta.yaml (hunk 1 above)
Comment (astamminger) [reply recipe/meta.yaml]: Unfortunately for python 3 the tests only worked for v1.8. I have no time right now, maybe we could put this on hold and discuss what would be the best solution next week?

Edit: I'll check the exact reason that makes the tests fail for <1.8 on python 3 on monday. Maybe it's safe to ignore.

Comment (ocefpaf) [reply recipe/meta.yaml]: conda will most likely install `django 1.8` in both cases anyway, that is why I'm puzzled by your selector here. Let me try something...

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
Let's try this one. If you ever need to fix this package just send a PR to this PR and build the build number.

File: recipe/meta.yaml (hunk 1 above)
Comment (astamminger) [reply recipe/meta.yaml]: Well this django-extensions version is used in a project we would like to add to conda-forge. However it also pins Django to version 1.7.11 (that's why I could not be too restrictive and used the selector)

Comment (ocefpaf) [reply recipe/meta.yaml]: > Well this django-extensions version is used in a project we would like to add to conda-forge. However it also pins Django to version 1.7.11 (that's why I could not be too restrictive and used the selector)

`<=1.8` is less restrict than `==1.8` for py3k and we can keep the package `noarch`.

Comment (astamminger) [reply recipe/meta.yaml]: Yeah, I meant pinning django==1.8 for all python versions.
